src/sudoers/ast.rs

```diff
@@ -418,11 +420,24 @@ impl Parse for MetaOrTag {
             "ROLE" | "TYPE" => unrecoverable!(
                 pos = start_pos,
                 stream,
-                "role based access control is not yet supported by sudo-rs"
+                "SELinux role based access control is not yet supported by sudo-rs"
             ),
 
+            #[cfg(feature = "apparmor")]
+            "APPARMOR_PROFILE" => {
+                expect_syntax('=', stream)?;
+                let StringParameter(profile) = expect_nonterminal(stream)?;
+                Box::new(move |tag| tag.apparmor_profile = Some(profile.clone()))
+            }
+
             "ALL" => return make(MetaOrTag(All)),
-            alias => return make(MetaOrTag(Alias(alias.to_string()))),
+            alias => {
+                if is_syntax('=', stream)? {
```

Comment:
Note: this addition is to act as a 'catch all' so we say "APPARMOR_PROFILE" is not supported (and other potential future THIS=THAT tags as well)